go/runtime/scheduling/simple/txpool/tester.go

```diff
@@ -87,9 +87,10 @@ func testBasic(t *testing.T, pool api.TxPool) {
 	require.EqualValues(t, 10, len(batch), "Batch size")
 	require.EqualValues(t, 51, pool.Size(), "Size")
 
-	hashes := make([]hash.Hash, len(batch))
-	for i, tx := range batch {
-		hashes[i] = tx.Hash()
+	hashes := make([]hash.Hash, 0, len(batch))
```

Comment:
Nit: Since you're inserting duplicates, should this be
```suggestion
	hashes := make([]hash.Hash, 0, 2*len(batch))
```
Or just remove the capacity entirely.